include/swift/AST/Module.h

```diff
@@ -648,27 +648,52 @@ class ModuleDecl : public DeclContext, public TypeDecl {
   /// \sa getImportedModules
   enum class ImportFilterKind {
     /// Include imports declared with `@_exported`.
-    Public = 1 << 0,
+    Exported = 1 << 0,
     /// Include "regular" imports with no special annotation.
     Private = 1 << 1,
```

Comment:
Would it be worth also renaming this case while we're here, to avoid confusion with `@_private` and `SourceFile::ImportFlags::PrivateImport`? Perhaps `Regular`?

Reply:
Yeah, I was in two minds about this... I was going to name it `Regular` or `Default` but neither of them felt sufficiently descriptive. @brentdax , @xymus : any naming suggestions for this one?

Reply:
Even if it's not very descriptive I like `Default`, it's clear enough as being for the default import behavior.

Reply:
`Default` is fine. `Interface` might work too (contrasting with `ImplementationOnly`).